src/components/Modals/NewCourse/CourseConflictModal.vue

```diff
@@ -187,32 +184,32 @@ export default defineComponent({
     resolveConflicts() {
       // edit the requirements assigned to the course when editor saved on reqs with changes made
       for (let i = 0; i < this.selectedReqsPerConflict.length; i += 1) {
```

Comment:
Why not for-each loop?

Reply:
Before it was implemented in a way that depended on `i`. Should be changed!